packages/styles/modes/intents/src/index.css

```diff
@@ -1,3 +1,5 @@
+@import url("@canonical/tokens/dist/css/semantic.css");
```

Comment:
What are your thoughts on export aliases to make this less dependent on file structure?